src/deluge/gui/views/arranger_view.cpp

```diff
@@ -1538,6 +1543,12 @@ void ArrangerView::editPadAction(int32_t x, int32_t y, bool on) {
 
 				if (x == xPressed && y == yPressedEffective) {
 
+					hid::display::OLED::clearMainImage();
+					ClipInstance* clipInstance = output->clipInstances.getElement(pressedClipInstanceIndex);
+					drawClipName(output, clipInstance->clip);
```

Comment:
should we be drawing clips on release? It's no longer selected at this point

Reply:
I tested some scenarios, this was the best to still view the latest selected clip. Otherwise, on the arranger its still difficult to see what's the last clip been added or changed.